operator-common/src/main/java/io/strimzi/operator/common/model/cruisecontrol/CruiseControlHeaders.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.common.model.cruisecontrol;
+
+/**
+ * Cruise Control headers.
+ */
+public class CruiseControlHeaders {
+    /**
+     * User task id.
+     */
+    public static final String USER_TASK_ID_HEADER = "User-Task-ID";
+}
```

Comment:
whatever it's a string within a class or an enum, I am not still convinced we need a dedicated class for this. How many headers do CC have for the REST API? AFAIU, just this one? More?
Tbh I would not remove it from the CruiseControlApi and not touch it until we have a clear plan where we want to go with the new client.

Reply:
You are the Cruise Control SME maintainer here ... so I think your _vote_ should matter more than mine 😉 

But just to make sure ... are you aware the same package has already many other Cruise Control constants defined there: https://github.com/strimzi/strimzi-kafka-operator/tree/main/operator-common/src/main/java/io/strimzi/operator/common/model/cruisecontrol? I personally would be against this if it was only this one constant living in the `operator-common` module. But as have already have many of them there, it seems like it matters lot less to me.

Reply:
Yeah fair enough.